### PR TITLE
Fix code examples in README

### PR DESCRIPTION
**Pull request by vyorkin**
Just a couple of fixes that I've made while reading:

* `Eff -> Effect`, `liftEff` -> `liftEffect`, etc
* `id` -> `identity`
* `eff :: EFF (..)` -> `effect :: FProxy Effect`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,8 +67,10 @@ exists as simple data. In order for it to do something, we'd need to provide
 an interpreter which pattern matches on the data types:
 
 ```purescript
+main :: Effect Unit
 main = foldFree go program
   where
+  go :: TalkF ~> Effect
```
Comment (vyorkin) [inline README.md:73]: its unable to infer the type otherwise

File: README.md [hunk 2]
```diff
@@ -307,28 +309,28 @@ all effects. Instead we can use `send` for unmatched cases.
...
   Listen reply -> do
     pure (reply "I am Groot")
 
 runTalk
-  :: forall r eff
-   . Run (eff :: EFF (console :: CONSOLE | eff), talk :: TALK | r)
-  ~> Run (eff :: EFF (console :: CONSOLE | eff) | r)
+  :: forall r
+   . Run (effect :: FProxy Effect, talk :: TALK | r)
+  ~> Run (effect :: FProxy Effect | r)
```
Comment (natefaubion) [inline README.md]: `EFFECT` is exported as an alias for `FProxy Effect`.

Comment (vyorkin) [reply README.md]: thanks, updated the PR to use `EFFECT`

File: README.md [hunk 3]
```diff
@@ -307,28 +309,28 @@ all effects. Instead we can use `send` for unmatched cases.
...
+  ~> Run (effect :: EFFECT | r)
 runTalk = interpret (on _talk handleTalk send)
 
-program2 :: forall eff r. Run (eff :: EFF (console :: CONSOLE | eff), dinner :: DINNER | r) Unit
+program2 :: forall r. Run (effect :: EFFECT, dinner :: DINNER | r) Unit
 program2 = dinnerTime # runTalk
 ```
 
-We've interpreted the `TALK` effect in terms of native `Eff` effects, and so
+We've interpreted the `TALK` effect in terms of native `Effect`'s, and so
```
Comment (natefaubion) [inline README.md]: I would say "in terms of the native `Effect` type" to avoid the awkward `'s`.

Comment (vyorkin) [reply README.md]: updated

**Review by natefaubion — APPROVED** [review]

**Comment by natefaubion** [comment]
Thanks!

**Pull request merged**